.github/CONTRIBUTING.md

```diff
@@ -37,8 +32,7 @@ Contributing to Issues
 * If the issue is marked as [`Up-for-Grabs`][up-for-grabs],
   the PowerShell Maintainers are looking for help with the issue.
```

Comment:
Sometimes it's not obvious fix, and it would be nice that maintainers accompany the comment how and where to fix, or at least where it's best to start.